src/components/ChecklistCard/ChecklistCard.jsx

```diff
@@ -6,9 +6,9 @@ import { Link } from 'react-router-dom'
 function ChecklistCard(props) {
   return (
       <>
-      <Link to={`/checklists/${props.checklist._id}`}>
+      <Link to={`/checklists/${props.checklists._id}`}>
           <div className={styles.container}>
-              <h2>{props.checklist.name}</h2>
+              <h2>{props.checklist.checklistName}</h2>
```

Comment:
Would change `checklistName` back to `name` unless `name` broke something for some reason.